### DISCUSSION TITLE
Get build tasks status

### DISCUSSION THREAD

**Discussion by aliakhlaghi93** [rfennell/AzurePipelines#1071] [Q&A] [answered]
Hi
Is it possible to get build tasks status like release tasks status?
I have tried to get all tasks logs but it is too messy. Do you know any better way?
Regards,
Ali

**Comment by rfennell** [comment]
Sorry not sure I understand what you mean. Do you the status of each task in a release pipeline?

If that is the case I think your only option is digging into the timeline, but you are right that is messy

**Reply by aliakhlaghi93** [reply]
Thank you for your reply.
I meant tasks in the build that is used as an artifact for the release pipeline.

**Comment by rfennell** [comment, marked as answer]
In a classic release there is an array of `build` objects that list the build and associated WI/CS.

There is no matching array in a build as there is only one 'build'. However, for multistage YAML pipelines there is a `consumedartfacts` array that provides some basic details of the consumed artficats by the pipeline.

Hope that helps

**Reply by aliakhlaghi93** [reply]
Thank you!
